### DISCUSSION TITLE
Has NLP been tested for ReDOS issues?

### DISCUSSION THREAD

**Discussion by MarketingPip** [nlp-compromise/nlp_compromise#1102] [Q&A]
@spencermountain -

Curious, I am sure you have battle tested things but just got an interesting vun report that made me think of Compromise that will affect performance of library.

**Comment by spencermountain** [comment]
yeah, we use the eslint-plugin-regexp library, which does some of that. https://github.com/spencermountain/compromise/blob/master/package.json#L110
Sure there's more we can do, though. help is welcome
